recipe/conda_build_config.yaml

```diff
@@ -1,7 +0,0 @@
-c_compiler_version:                       # [osx]
-  - 13                                    # [osx]
-cxx_compiler_version:                     # [osx]
-  - 13                                    # [osx]
-llvm_openmp:                              # [osx]
-  - 13                                    # [osx]
```

Comment:
The old compiler pin was the issue. We split the libcxx headers off starting from v16 now, but haven't backported this to older compilers (mainly because it's a lot of [work](https://github.com/conda-forge/clangdev-feedstock/issues/310#issuecomment-2330421676)).

It's still possible to compile stuff with old compilers though, the only thing necessary then is to add `libcxx-devel` in build/host.

Reply:
Ahhh great. Thanks for confirming @h-vetinari!